src/main/java/com/esir/jxc/mpc/repository/ArticleAddedRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.esir.jxc.mpc.repository;
+
+import com.esir.jxc.mpc.model.ArticleAdded;
+import org.springframework.data.domain.Page;
+import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ArticleAddedRepository extends ElasticsearchRepository<ArticleAdded, String> {
```

Comment:
```suggestion
public interface ArticleRepository extends ElasticsearchRepository<ArticleAdded, String> {
```
➜ vous n'allez quand même pas créer un repo par type d'event